src/components/Card.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import styled from 'styled-components'
+
+
+function Card({ data }) {
+
+	return (
+		<div>
+			{data.name}
+		</div>
+	)
+}
+
+export default Card
```

Comment:
Add eslint to the project and add the rule that will detect and add new empty line